### PR TITLE
 handle get profile artist route

### PR DESCRIPTION
**Pull request by AlaaSaadeddin**
-  handle get profile artist route
relates #21

### CODE REVIEW

File: server/database/queries/Artist/getArtistQuery.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const connection = require('../../connection');
+
+const getArtistQuery = (id) =>
+  connection.query('SELECT * FROM artist WHERE id = $1', [id]);
```
Comment (ranasobeid95) [inline server/database/queries/Artist/getArtistQuery.js:4]:
```suggestion
  connection.query({
    text: 'SELECT * FROM artist WHERE id = $1',
    values: [id],
  });
```

File: server/controllers/getArtist.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const { getArtistQuery } = require('../database/queries');
+
+const getArtist = async (req, res, next) => {
+  try {
+    const { artistId } = req.params;
+    if (artistId > 0) {
+      const { rows } = await getArtistQuery(artistId);
+      const data = { ...rows[0] };
```
Comment (ranasobeid95) [inline server/controllers/getArtist.js:8]: You can use destructuring to get object directly
```suggestion
       const {
        rows: [data],
      } = await getArtistQuery(artistId);
```

File: server/controllers/getArtist.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const { getArtistQuery } = require('../database/queries');
+
+const getArtist = async (req, res, next) => {
+  try {
+    const { artistId } = req.params;
```
Comment (ranasobeid95) [inline server/controllers/getArtist.js:5]:
```suggestion
const {
      params: { artistId },
    } = req; 
```

File: server/database/queries/Artist/getArtistQuery.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const connection = require('../../connection');
+
+const getArtistQuery = (id) =>
+  connection.query({
+    text: 'SELECT * FROM artist WHERE id = $1',
```
Comment (LinaYahya) [inline server/database/queries/Artist/getArtistQuery.js:5]: try not to select * , keep password out

File: server/routes/index.js [hunk 5]
```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
 
 const paintings = require('./paintings');
+const Artist = require('./artist');
```
Comment (LinaYahya) [inline server/routes/index.js:4]: It's better to be lowercase to save consistency

**Pull request merged**